R/processing_pcs.R

```diff
@@ -126,3 +126,47 @@ calculate_within_batch_ari_pcs <-
 
     return(within_batch_ari_tibble)
   }
+
+#' Calculate silhouette width scores from a merged SCE object.
```

Comment:
Same comment from #199 to move this into the same script where you have the wrapper function.